models/response_entities.go

```diff
@@ -80,10 +80,17 @@ type AttachmentLinksResource struct {
 	Download string `json:"download"`
 }
 
+// ResolutionResource contains the details of the resolution resource
 type ResolutionResource struct {
 	DateOfResolution string `json:"date_of_resolution"`
 }
 
+// StatementOfAffairsResource contains the details of the statement of affairs resource
+
+type StatementOfAffairsResource struct {
```

Comment:
We should probably add this to the Insolvency response struct as well, so that if we need to query the entire Insolvency object from the database (to send to CHIPS, for example), then we also have the SOA data being returned as part of that Insolvency response